### PR TITLE
Fix issue at 18.06.2023

### PR DESCRIPTION
**Pull request by AndriyStechenko**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -38,12 +38,14 @@ function deleteDicesFromMemory () {
 
 function clearDiceContainer() {
     const diceContainer = document.getElementById("dice-container");
-    while (diceContainer.firstChild) {
-      diceContainer.removeChild(diceContainer.firstChild);
+    const diceChildren = diceContainer.childNodes;
+    
+    for (let i = diceChildren.length - 1; i >= 0; i--) {
```
Comment (bmalets) [inline js/app.js]: do not give names `i`, `y`, `z` to the variables.

File: js/models/scoreTable.js [hunk 2]
```diff
@@ -22,7 +23,8 @@ export default class ScoreTable {
         this.secondPlayerTurnsHistory = null
     }
 
-    setPlayerName (name) {
-        this.firstPlayer = name
+    setPlayerName () {
+        this.firstPlayer = readFromStorage('player')
+        console.log('players name from class:' ,this.firstPlayer)
```
Comment (bmalets) [inline js/models/scoreTable.js]: space must go after of comma
try to use js lint https://www.codeguru.com/tools/the-top-five-javascript-linting-tools/

File: js/modules/scoreTableInitialization.js [hunk 3]
```diff
@@ -33,8 +31,8 @@ function setComputerNameInTable () {
     playerNameCell.textContent = currentPlayerName.secondPlayer
 }
 
-let sum
 function scoreTableResultSum() {
+    let sum
     let turnScore = readFromStorage('currentTurn')
       sum = 0;
```
Comment (bmalets) [inline js/modules/scoreTableInitialization.js]: why `sum = 0;` has extra 2 spaces in front?

Comment (bmalets) [reply js/modules/scoreTableInitialization.js]: why this line has `;` in the end but another lines does not?

File: js/modules/scoreTableInitialization.js [hunk 4]
```diff
@@ -114,7 +106,7 @@ function createResultTable() {
     const headRow = document.createElement('tr');
   
     let existingScoreTable = readFromStorage('currentScoreTable');
-    let headers = ['#', existingScoreTable.firstPlayer, existingScoreTable.secondPlayer, 'Player#3'];
+    let headers = ['#', existingScoreTable.firstPlayer, existingScoreTable.secondPlayer];
```
Comment (bmalets) [inline js/modules/scoreTableInitialization.js]: what is `existingScoreTable.firstPlayer`? is it an object of the class Player or just a String with player's name?

**Review by bmalets — CHANGES_REQUESTED** [review]
Please fix all things that I have mentioned in separate commits and give an answers/emojis to each comment 🙂

File: js/modules/scoreTableInitialization.js (hunk 4 above)
Comment (AndriyStechenko) [reply js/modules/scoreTableInitialization.js]: existingScoreTable.firstPlayer  -   it's part of currentScoreTable obj what was made on the base of Player class and info, and  get name as from it

Comment (bmalets) [reply js/modules/scoreTableInitialization.js]: I understand that. These variable `headers` show data in the HTML table.
So why do you need a player object here, not just a string with the player name?

**Review by bmalets — APPROVED** [review]

**Pull request merged**